dora/tests/testcontainers/src/test/java/alluxio/membership/MembershipManagerTest.java

```diff
@@ -401,4 +401,44 @@ public void testSameWorkerIdentityConflict() throws Exception {
     Assert.assertTrue(curWorkerInfo.isPresent());
     Assert.assertEquals(wkr2.getAddress(), curWorkerInfo.get().getAddress());
   }
+
+  @Test
+  public void testWorkerHttpServerPorts() throws Exception {
```

Comment:
can u say testOptionalFieldChangeInWorkerAddress

Reply:
Done. Updated the function name to `testOptionalHttpPortChangeInWorkerAddress`.